build.sbt

```diff
@@ -364,7 +365,8 @@ lazy val root: Project = Project("scio", file("."))
     scioExamples,
     scioRepl,
     scioJmh,
-    scioMacros
+    scioMacros,
+    scioSmb
```

Comment:
should i remove this until the Scio/Scala bindings are completed? should be a fast follow (before we publish the next Scio version, anyway)

Reply:
another way to go about this would be to create a 0.9 branch and have this there.

Reply:
It doesn't have to be in 0.9 though since it's new features not breaking changes :)

Reply:
hm true, but was under the impression this was not for 0.8? or is it?

Reply:
It may not be ready in time for 0.8.0 release but don't see why it can't get released in 0.8.1 :) happy to move it to an 0.8.1 branch if that's the case, @nevillelyh / @regadas wdyt?